src/utils/positionUtils.ts

```diff
@@ -281,13 +295,16 @@ export const calcTradeFee = ({
       Big6Math.mul(makerFee.proportionalFee, adjustedMakerTotal),
       makerFee.scale,
     )
+    // NOTE: worth double checking if this is the correct return value
     const makerProportionalFee = Big6Math.mul(notional, makerProportionalFeeRate)
     const makerLinearFee = Big6Math.mul(notional, makerFee.linearFee)
     const total = makerLinearFee + makerProportionalFee
     const feeBasisPoints = !Big6Math.isZero(total) ? Big6Math.div(total, notional) : 0n
 
     tradeFeeInfo = {
       total,
+      tradeImpact: 0n,
```

Comment:
For makers there is no priceImpact so the `tradeFee` and the `total` are the same value